### DISCUSSION TITLE
Is the Remix.run website source open-source?

### DISCUSSION THREAD

**Discussion by dodas** [remix-run/remix#3663] [Q&A] [answered]
Is the code for the [remix.run](https://remix.run/) website available publicly?
I only found the [docs files](https://github.com/remix-run/remix/tree/main/docs), but couldn't find the source code for the website itself.

Thanks!

**Comment by kiliman** [comment]
No, unfortunately not.

**Comment by sergiodxa** [comment]
No, and when I asked the team they mentioned to have no plans to open source it since that will create a new inbox to review (issues, PRs, discussions).

**Reply by kiliman** [reply]
Ha. Plus they deserve to at least keep some secret sauce for themselves.

**Comment by dodas** [comment]
Gotcha, thanks.

I would like to build a Docs site with mdx content similar to Remix's one.
Anyone got some boilerplate or gist of how to set this up with things like auto-generated navigation, table of contents etc?

**Reply by stereokai** [reply]
Have you had any progress with this? Cheers!

**Comment by HananoshikaYomaru** [comment, marked as answer]
it is now open source https://github.com/remix-run/remix-website
